signbank/dictionary/csv_import.py

```diff
@@ -441,7 +441,7 @@ def confirm_import_nzsl_share_gloss_csv(request):
                     video_url = gloss_data["videos"]
                     extension = video_url[-3:]
                     file_name = (
-                        f"{gloss.pk}-{gloss.idgloss}_video.{extension}"
+                        f"{gloss.pk}-{word_en}.{gloss.pk}_video.{extension}"
```

Comment:
**question**: if the same format is being used for filenames, now that there are more parts to it, might it be worth extracting into a function (potentially a method on the gloss?)

Reply:
this file name format is slightly different from the one on the model's `save` method. I found the colon in the idgloss (which is just `word:pk`) caused issues with AWS.